### PR TITLE
[Owner] Tests for Initialize session , Close session , getting error_message for DCPower

### PR DESCRIPTION
**Pull request by ni-pkhare**
### Justification

Session tests involving 3 APIs :
1) Initialize_with_independent_channels
2) Close
3) ErrorMessage

Task Link : https://ni.visualstudio.com/DevCentral/_workitems/edit/1367881

### Implementation

Tests:
1) Initializing session with device and session name.
2) Initializing session with device and no session name.
3) Initializing session without device (ie. option_string = "").
4) Closing driver session.
5) Closing driver session with no error reported.
6) Getting error message when no device is present.

### Testing
Manually ran the system tests.

### CODE REVIEW

File: source/tests/system/nidcpower_session_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+namespace dcpower = nidcpower_grpc;
+
+const int kViErrorRsrcNFound = -1073807343;
+const char* kViErrorRsrcNFoundMessage = "VISA:  (Hex 0xBFFF0011) Insufficient location information or the device or resource is not present in the system.";
+const char* kTestResourceName = "FakeDevice";
+const char* kSimulatedOptionsString = "Simulate=1, DriverSetup=Model:4147; BoardType:PXIe";
+const char* kTestSessionName = "SessionName";
+const char* kInvalidResourceName = "";
+
+class NiDCPowerSessionTest : public ::testing::Test {
```
Comment (kruthi-NI) [inline source/tests/system/nidcpower_session_tests.cpp:19]: @ni-pkhare , you need to make changes in cmakelists.txt to add this file to systemtestsrunner

Comment (ni-pkhare) [reply source/tests/system/nidcpower_session_tests.cpp:19]: Yes, made the necessory changes.

**Review by kruthi-NI — APPROVED** [review]

**Review by ganeshnie — APPROVED** [review]

**Review by reckenro — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -300,6 +300,7 @@ add_executable(SystemTestsRunner
     "source/tests/system/niswitch_driver_api_tests.cpp"
     "source/tests/system/niswitch_session_tests.cpp"
     "source/tests/system/nisync_session_tests.cpp"
+    "source/tests/system/nidcpower_session_tests.cpp"
```
Comment (SparkingSpork) [inline CMakeLists.txt]: I'd like to keep the driver test files in alphabetical order. That is:
```
    "source/tests/system/session_utilities_service_tests.cpp"
    "source/tests/system/nidcpower_session_tests.cpp"
    "source/tests/system/niscope_driver_api_tests.cpp"
    "source/tests/system/niscope_session_tests.cpp"
    "source/tests/system/niswitch_driver_api_tests.cpp"
    "source/tests/system/niswitch_session_tests.cpp"
    "source/tests/system/nisync_session_tests.cpp"
```

**Review by SparkingSpork — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (ni-pkhare) [reply CMakeLists.txt]: Done.

**Pull request merged**